pytorch_binding/setup.py

```diff
@@ -8,10 +8,10 @@
 
 
 extra_compile_args = ['-fPIC']
-if LooseVersion(torch.__version__) >= LooseVersion("1.5.0"):
-    extra_compile_args += ['-std=c++14']
+if LooseVersion(torch.__version__) >= LooseVersion("2.1.0"):
+    extra_compile_args += ['-std=c++17']
 else:
-    extra_compile_args += ['-std=c++11']
+    extra_compile_args += ['-std=c++14']
```

Comment:
What about PyTorch versions less 1.5.0? Earlier c++11 used for these, now will be used c++14, is it ok?

Reply:
PyTorch version less 1.5.0 cannot run in my local platform. Since it is an old version, can we just use C++17 to cover all PyTorch version?